Experimental/CameraComponent/Camera.cs

```diff
@@ -1,24 +1,12 @@
-﻿using System;
-using System.Collections.Generic;
-using System.ComponentModel;
-using System.Linq;
-using System.Numerics;
-using System.Text;
-using System.Threading.Tasks;
-using Windows.UI.Composition;
-using Windows.UI.Composition.Scenes;
-using Windows.UI.Xaml.Media;
-
+﻿using Windows.UI.Composition;
 
 namespace CameraComponent
 {
-    public interface Camera : INotifyPropertyChanged
+    public interface Camera
     {
         Projection Projection { get; set; }
-        Matrix4x4 CreateViewMatrix();

```

Comment:
I wonder if it is still useful to have a public method that is able to return the matrix that the viewport/camera creates.